rpm/SPECS/tapir-cli.spec

```
@@ -0,0 +1,57 @@
+Name:          tapir-cli
```

RuboCop (1.75.5)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)